containers/python/python_container_entry.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
 
-conda env create -f=/model/environment.yml -n container-env
-/bin/bash -c "source activate container-env && exec python /container/python_container.py"
+conda install -y --file /model/conda_dependencies.txt
+pip install -r /model/pip_dependencies.txt
+/bin/bash -c "exec python /container/python_container.py"
```

Comment:
So I had another idea about how to speed things up. Can you first try running the container without installing any of the supplied dependencies. If that fails, then install the dependencies and try running it again. So change `python_container_entry.sh` to something like:

```sh
/bin/bash -c "exec python /container/python_container.py
# if error...
conda install -y --file /model/conda_dependencies.txt
pip install -r /model/pip_dependencies.txt
/bin/bash -c "exec python /container/python_container.py"
```